packages/frontend/web-ui/src/app/store/features/authSlice.js

```diff
@@ -1,6 +1,35 @@
 import { createSlice } from '@reduxjs/toolkit';
+import { createAuthByCredentials } from '../thunk/createAuthByCredentials';
 import { createAuthByToken } from '../thunk/createAuthByToken';
 
+function createAuthPendingReducer(state) {
+  state.status = 'checking';
```

Comment:
should we declare a constant for this string value?

Reply:
Done! The name of this new const variable is **STATUS_AUTH_CHECKING**.